frontend/src/css/globals.css

```diff
@@ -40,11 +40,11 @@
     --secondary: 210 40% 96.1%;
     --secondary-foreground: 222.2 47.4% 11.2%;
 
-    /* tailwind blue-500 */
-    --accent: 217 91% 60%;
+    /* blue-3 */
+    --accent: 209 100% 96.5%;
+    /* blue-12 */
+    --accent-foreground: 216 71% 23%;
```

Comment:
@akshayka, i think this looks better (lighter background on darker foreground), but let me know if you prefer the other way

Reply:
we can chat live about this one and cycle through the scale together

Reply:
ive reverted the accent change - but the accent probably shouldnt be dark. i think this color got switched up at some point and we should instead make `primary` our dark blue. ill handle this in a follow up